_data/faqs.yaml

```diff
@@ -1,76 +1,76 @@
+    Development of the repository application began in March 2016 after an intensive Design Phase was undertaken and produced a collection of <a href="https://wiki.duraspace.org/display/hydra/Hydra-in-a-Box+Design+Documents">design documents</a>. A series of project team work cycles and community-based sprints now follow, leading up to several pilots planned for 2017. Watch our <a href="http://hydrainabox.projecthydra.org/blog.html">blog</a> or <a href="https://www.youtube.com/channel/UC1eopKtm-k_f93RTfzHSvMQ">demo videos</a> for updates along the way. More information about the 2017 pilots and service roll-out will be posted here in early 2017.
 - question: >-
-    Who will provide hosting services for Hydra-in-a-Box?
+    What kinds or sizes of institutions is Hyku intended for? Can I use Hyku if my institution has minimal IT staff with no skills in Ruby programming?
   answer: >-
-    DuraSpace will provide hosting services at launch time, and we’ve been speaking with other hosting service providers about their needs and requirements. As hosted service options become clear, we will definitely let everyone know!
+    Hyku will be useful to any institution of any size managing collections of digital assets, particularly libraries, archives, museums, and historical societies with collections of cultural heritage materials. We believe, and IMLS supports us in this belief, that all institutions should have access to the benefits of the Hydra community and its products without needing to hire and retain Ruby developers. Our goals are to produce a repository application that is easy to install and easy to maintain over time, and to provide software that is compelling to hosted service providers.  In these ways, Hyku will be a solution for any institution seeking a next-generation digital repository solution for its digital collections, particularly if technical resources and infrastructure are limited.
 - question: >-
-    Is the Hydra-in-a-Box application dependent on Amazon Web Services? What if I want to host it on a different cloud service provider, or on local infrastructure?
-  answer: >-
-    No, the application is not at all tied to AWS. If you choose to run it on AWS, take advantage of the CloudFormation templates our project team developed (<a href="https://github.com/hybox/aws">available on GitHub</a>). If you choose to install it on another cloud service provider, you may need to do similar development for that environment. For local installations, you need not pay attention to any of this template business.
+    What is the relationship between Hyku, Hyrax, and Sufia? 
+  answer: >- 
+    Hyku is based directly on Hyrax. Hyku had been based on Sufia until November 2016 when Sufia and CurationConcerns were consolidated into one gem named Hyrax. From Hyku's perspective, there has been no functional change since Sufia was consolidated into Hyrax.
```

Comment:
Thanks for the close review, @mjgiarlo. I addressed everything but for the outstanding question to Justin.